test/acceptance/features/bindAppToService.feature

```diff
@@ -269,9 +270,171 @@ Feature: Bind an application to a service
             status:
                 ready: true
             """
-        Then Secret "binding-request-backend" contains "CustomReady" key with value "true"
+        Then Secret should be updated
+        Then Secret contains "CustomReady" key with value "true"
 
 
+    Scenario: Backend Service status update gets propagated to the binding secret for Volume Mount
```

Comment:
Why is this part of this changelist? Can't this be moved to another more relevant changelist, or there are fixes for this specific situation in this PR?